R/tf_estimator.R

```diff
@@ -251,12 +254,25 @@ evaluate.tf_estimator <- function(object,
 #' @family custom estimator methods
 export_savedmodel.tf_estimator <- function(object,
                                            export_dir_base,
-                                           serving_input_receiver_fn,
+                                           serving_input_receiver_fn = NULL,
                                            assets_extra = NULL,
                                            as_text = FALSE,
                                            checkpoint_path = NULL,
                                            ...)
 {
+  if (is.null(serving_input_receiver_fn)) {
+    if ("tf_custom_estimator" %in% class(object))
+      stop("A 'tf_custom_estimator' requires a custom `serving_input_receiver_fn`.")
+    
+    input_spec <- regressor_parse_example_spec(
```

Comment:
@javierluraschi Sorry I overlooked this. Seems like here you are only using `regressor_parse_example_spec` but we should check for "regressor" or "classifier" and then choose a parser based on that. Could you fix this as well as adding a test case when you get a chance?

Reply:
@javierluraschi Just a reminder to make sure this is somewhere on your todo list (not urgent we do this right away just don't want to lose track of it)